PWGDQ/Tasks/tableReader_withAssoc.cxx

```diff
@@ -1178,6 +1179,8 @@ struct AnalysisSameEventPairing {
   std::map<int, std::vector<TString>> fMuonHistNames;
   std::map<int, std::vector<TString>> fTrackMuonHistNames;
   std::vector<AnalysisCompositeCut> fPairCuts;
+  std::vector<AnalysisCompositeCut> fTrackCuts;
```

Comment:
Here you create a vector of composite cuts, but you actually just need the name of the cuts. So I suggest to make a vector of TString only.